app/index.html

```diff
@@ -9,6 +9,8 @@
 
   <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
   <link rel="stylesheet" href="{{rootURL}}assets/open-event-frontend.css">
+  <link rel="preload" href="https://eventyay.com/fonts/lato-v17-latin-300.woff2" as="font" crossorigin="anonymous">
+  <link rel="preload" href="https://eventyay.com/fonts/lato-v17-latin-regular.woff2" as="font" crossorigin="anonymous">
```

Comment:
What is eventyay.com?

Reply:
> What is eventyay.com?

@iamareebjamal I copy the address of the error which is showing in preload key request

Code syantx I used-
<link rel="preload" href="Font URL Showing in PageSpeed" as="font" crossorigin="anonymous">

Reply:
And I used this syantx code to solved this issue
<link rel="preload" href="Font URL Showing in PageSpeed" as="font" crossorigin="anonymous">

Reply:
But this is open event project, not eventyay

Reply:
> But this is open event project, not eventyay 

I will fixed